### PR TITLE
Profile screen for non-users

### PR DESCRIPTION
**Pull request by chrisnojima**
Supports non-user screens in profile
Search for 'asd' on github to see a profile

### CODE REVIEW

File: shared/actions/tracker2.js [hunk 1]
```diff
@@ -143,6 +144,11 @@ function* load(state, action) {
       })
     )
   } catch (err) {
+    if (err.code === RPCTypes.constantsStatusCode.scresolutionfailed) {
```
Comment (chrisnojima) [inline shared/actions/tracker2.js]: mark as not a user

File: shared/common-adapters/reload.js [hunk 2]
```diff
@@ -123,10 +124,14 @@ export type OwnProps = {|
   reloadOnMount?: boolean,
   title?: string,
   waitingKeys: string | Array<string>,
+  errorFilter?: RPCError => boolean,
```
Comment (chrisnojima) [inline shared/common-adapters/reload.js]: allow reloadable to skip some errors

**Comment by chrisnojima** [comment]
@keybase/react-hackers this is ready

**Comment by chrisnojima** [comment]
there isn't a convenient way to get the proof icons so i deferred that

File: shared/profile/user/container.js [hunk 3]
```diff
@@ -30,15 +30,18 @@ const mapStateToProps = (state, ownProps) => {
...
+  const followersCount =
+    (!notAUser && state.tracker2.usernameToDetails.getIn([username, 'followersCount'])) || 0
+  const followingCount =
+    (!notAUser && state.tracker2.usernameToDetails.getIn([username, 'followingCount'])) || 0
 
   return {
     _assertions: d.assertions,
     _suggestionKeys: userIsYou ? state.tracker2.proofSuggestions : null,
-    backgroundColorType: headerBackgroundColorType(d.state, followThem),
+    backgroundColorType: notAUser ? 'blue' : headerBackgroundColorType(d.state, followThem),
```
Comment (buoyad) [inline shared/profile/user/container.js]: :sheep: `headerBackgroundColorType` could do this

File: shared/profile/user/index.js [hunk 4]
```diff
@@ -98,9 +100,22 @@ const Proofs = p => {
...
 
+  let proveIt = null
+
+  if (p.notAUser) {
+    const [name, service] = p.username.split('@')
+    proveIt = (
+      <Kb.Text
+        type="BodySmall"
+        style={styles.proveIt}
+      >{`Tell ${name} to join Keybase and prove their ${service}.`}</Kb.Text>
```
Comment (buoyad) [inline shared/profile/user/index.js]: don't need the `{``}` around this

File: shared/profile/user/index.js [hunk 5]
```diff
@@ -536,6 +558,9 @@ export const styles = Styles.styleSheetCreate({
       paddingRight: Styles.globalMargins.medium,
     },
   }),
+  proveIt: {
```
Comment (buoyad) [inline shared/profile/user/index.js]: gap `Box2` above instead

Comment (chrisnojima) [reply shared/profile/user/index.js]: that puts gaps between the proofs which we don't want (i tried that first)

**Pull request merged**